src/masternodes/rpc_proposals.cpp

```diff
@@ -100,15 +100,44 @@ UniValue proposalToJSON(const CProposalId &propId,
     return ret;
 }
 
-UniValue proposalVoteToJSON(const CProposalId &propId, uint8_t cycle, const uint256 &mnId, CProposalVoteType vote) {
+UniValue proposalVoteToJSON(const CProposalId &propId, uint8_t cycle, const uint256 &mnId, CProposalVoteType vote, bool valid) {
     UniValue ret(UniValue::VOBJ);
     ret.pushKV("proposalId", propId.GetHex());
     ret.pushKV("masternodeId", mnId.GetHex());
     ret.pushKV("cycle", int(cycle));
     ret.pushKV("vote", CProposalVoteToString(vote));
+    ret.pushKV("valid", valid);
     return ret;
 }
 
+struct VoteAccounting {
+    int totalVotes;
+    int yesVotes;
+    int neutralVotes;
+    int noVotes;
+    int unknownVotes;
```

Comment:
Is it really unknown here, can't we be sure that they are invalid ?

Reply:
This is just to handle this case in `CProposalVoteToString`

https://github.com/DeFiCh/ain/blob/314cf76e9917b345e107853da9240a2baca07176/src/masternodes/proposals.cpp#L48

This is actually a change from https://github.com/DeFiCh/ain/pull/1714

Reply:
Removed in #1714